controller/volume_controller.go

```diff
@@ -2494,7 +2494,7 @@ func (c *VolumeController) getIsSchedulableToDiskNodes(v *longhorn.Volume, nodeN
 			// TODO: record the message to condition
 			log.Warn("Found 0 node has at least one schedulable disk")
 		} else {
-			log.Debugf("Found node %v has at least one schedulable disk", schedulableNodeNames)
+			log.Infof("Found node %v has at least one schedulable disk", schedulableNodeNames)
```

Comment:
Do you think we should log this info or debug? Doesn't it mean nodes have schedulable disks if there is no `Found 0 node has at least one schedulable disk` logged? probably we can keep this debug to prevent messages, because it seems this is related to replica rebuilding. WDYT?

Reply:
Sure. Updated. Thank you.